src/api/models/Administrator.ts

```diff
@@ -0,0 +1,13 @@
+import {UserStatus} from './User';
+
+
+export type Administrator = {
```

Comment:
Administrator is a variant of a User so I think it would be better to build a relation:

```
export type Administrator = Pick<User, 'createTime' | 'deleteTime' | 'email' | 'id' | 'status'> & {
  isAdmin: boolean;
  isAdminReadOnly: boolean;
  isUserAdmin: boolean;
}
```